### DISCUSSION TITLE
Provider AWS is not supported. Please change Provider "AWS" to "aws"

### DISCUSSION THREAD

**Discussion by Tsingis** [serverless/serverless#12602] [Q&A] [answered]
Getting _Provider AWS is not supported. Please change Provider "AWS" to "aws"_ when running `serverless deploy`

_serverless.yml_
```
provider:
  name: aws
```
What exactly is wrong here? Using serverless 4.1.4

**Comment by wilr** [comment]
We had a similar issue and it was with a custom variable not the provider name

```
authorizer-auth0:
  Fn::Sub: "arn:aws:lambda:**${AWS::Region}**
```

**Reply by Tsingis** [reply, marked as answer]
Ok got it too. In similar situation I replaced `!Sub ${AWS::AccountId}` with `${aws:accountId}`

**Comment by et304383** [comment]
What an incredibly obtuse error message.  These variables are not even meant to be serverless variables.  They're supposed to be ignored.

I literally cannot change this one:

```!Sub arn:${AWS::Partition}:sqs:${aws:region}:${aws:accountId}:lambda-automation-errors```

Serverless still has no variable for the partition.

**Comment by Tsingis** [comment]
@et304383 The newest release answer to your problem? https://github.com/serverless/serverless/releases/tag/v4.1.13 with support to pseudoparameters.
